x-pack/plugins/security_solution/public/management/common/routing.ts

```diff
@@ -178,3 +179,15 @@ export const getTrustedAppsListPath = (location?: Partial<TrustedAppsListPageLoc
     querystring.stringify(normalizeTrustedAppsPageLocation(location))
   )}`;
 };
+
+export const getEventFiltersListPath = (
+  location?: Partial<TrustedAppsListPageLocation>
+): string => {
+  const path = generatePath(MANAGEMENT_ROUTING_EVENT_FILTERS_PATH, {
+    tabName: AdministrationSubTab.eventFilters,
+  });
+
+  return `${path}${appendSearch(
+    querystring.stringify(normalizeTrustedAppsPageLocation(location))
```

Comment:
Same comment for `normalizeTrustedAppsPageLocation` as before

Reply:
Yeah, my mistake. Will correct it.